chess_env.py

```diff
@@ -18,20 +18,32 @@ def reset(self, state=None):
         """
         pass
 
-    def step(self, move):
+    def step(self, move): #move assumed to be in the following format: move = chess.Move.from_uci("g1f3")
         """
         Make a move in the environment.
         :param move:
         :return:
         """
-        pass
 
-    def get_possible_moves(self):
+        if move not in self.legal_moves: #check if the move is legal
+            print("This move is not a legal move")
+        else:
+            self.push(move)
+            print(self)
```

Comment:
We shouldn't be printing every time we make a move. If you'd like to log it, you can use a logger like: https://realpython.com/python-logging/.